types/common/KeyReference.raml

```diff
@@ -0,0 +1,12 @@
+#%RAML 1.0 DataType
+(package): Common
+(docs-uri): https://docs.commercetools.com/http-api-types.html#keyreference
+
+displayName: KeyReference
+type: object
+discriminator: typeId
+properties:
+  typeId:
+    type: string
```

Comment:
Why do we use `string` here? Why not `ReferencetypeId`?

Reply:
Good catch,  it was a mistake.